adapters/visx/visxtest/params/race/banner.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I believe you need to specify sample test params here rather than keeping it empty since you do expect Visx specific params to be sent in the request

Reply:
Added them

Reply:
Thanks! Also, looks like there's some new conflicts unfortunately. Can you please resolve them?